build-support/bin/_release_helper.py

```diff
@@ -854,6 +854,9 @@ def build_pex(fetch: bool) -> None:
         Path(tmpdir, "BUILD_ROOT").touch()
         # We also need to filter out Pants options like `PANTS_CONFIG_FILES`.
         env = {k: v for k, v in env.items() if not k.startswith("PANTS_")}
+        # Disable the Pants repository-internal internal_plugins.test_lockfile_fixtures plugin because
+        # otherwise inclusion of that plugin will fail due to its `pytest` import not being included in the pex.
+        env["PANTS_BACKEND_PACKAGES"] = '-["internal_plugins.test_lockfile_fixtures"]'
```

Comment:
@stuhood / @Eric-Arellano: This is the fix for the wheel-building failure. Thoughts on whether this is a "correct" fix?

Reply:
So, it seems like the issue is that this plugin has dependencies which are not included in the `pants` binary, and so when pants has been packaged, they fail to import.

That sounds right. But on the other hand, this comment indicates that we're intentionally loading the plugins: https://github.com/pantsbuild/pants/blob/8dd95105ad5aa917034786cbf8146b5715431a45/build-support/bin/_release_helper.py#L285-L288 ... I guess so that we don't have to use `--no-verify-config` (which might mask errors?).

So: yea, this seems fine. But I would extract the two usecases into a single provider of the env with a single comment.

Reply:
Refactored it into a constant.